Dockerfile

```diff
@@ -2,8 +2,8 @@
 # This Dockerfile requires BuildKit to be enabled, by setting the environment variable
 # DOCKER_BUILDKIT=1
 # see https://docs.docker.com/build/buildkit/#getting-started
-ARG BASE_DIGEST_AMD64="sha256:d1c582c712e8ae049014a3e81296c522360949cbac7ade7e50a0d4d310706066"
-ARG BASE_DIGEST_ARM64="sha256:dac99a547dae7b46de7f0c34f57002094e054724058e8c9f51243d53286bf89e"
+ARG BASE_DIGEST_AMD64="sha256:9702b2b5cdba1aafb1544b5e2de58f6d2b3d6ed4f261b56f5cf479830d74aea4"
```

Comment:
I needed to update the amd64 digest here at the same time, to make sure the new inherited label is present on both images.

Reply:
If we override the inherited label with our version then we can omit this in the future. We could also make the labels check less strict to allow for inherited labels, but I'm actually a little wary about that, in case we would inherit strange things without knowing :smile:

Reply:
I agree to not make the check more loose but rather handle these labels.